### DISCUSSION TITLE
Feedback on AA Mtg List use on recoverydharma.org

### DISCUSSION THREAD

**Discussion by furick1** [code4recovery/12-step-meeting-list#1382] [Show and tell]
I recently began RD and am volunteering as well. I was tapped to help fix mtg mgmt and the website. It was clear after some research using this plugin was the way to go, especially since the site ran on WP.  Thanks to the 12 step meeting list team who created and maintained this with the community. It truly is going to help ease non profit management of RD mtgs and help people find meetings which is the real win.

The site use a devi theme and some custom work done by others. I am open to constructive feedback on the [implementation](https://recoverydharma.org/meetings/) of the plugin and anything else Wordpress/site relevant that could help the community.

Thanks again!

**Comment by dougie-wougie** [comment]
Nice site. I built all the AA sites for which I am the webservant in Divi. Bonus: you are in Portland. I used to live there with my daughter for five years; now, I am back in LA. I know exactly where you are located. It may be a good resource for my daughter since she thinks A.A. is only for old people like me. Lol.

**Comment by joshreisner** [comment]
nice job! i helped out with the refuge recovery website many years ago so it's great to see the plugin serving this community again

looks like the appearance on mobile could be an area of focus going forward - there's a gap to the left and right of the list caused by a css rule coming from the theme `.container { width: 80%; }` - would be great to get this edge-to-edge to use the full real estate on small screens. 

also if there's a way to get the meeting list to appear above the fold that would help get orient users on the page
